### DISCUSSION TITLE
pkg(8) --debug verbosity increases with -dd, -ddd and -dddd

### DISCUSSION THREAD

**Discussion by grahamperrin** [helloSystem/hello#383] [Show and tell]
Spun off from <https://github.com/helloSystem/ISO/issues/374#issuecomment-1236169494> and from today's <https://github.com/helloSystem/ISO/issues/404#issuecomment-1256928608>: 

* FreeBSD bug [266581 – ports-mgmt/pkg: pkg(8) should mention --debug verbosity increases with -dd, -ddd and -dddd](https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=266581)

**Comment by probonopd** [comment]
This is useful when one needs to find out why a certain package is being installed.

**Reply by grahamperrin** [reply]
Debugging is powerful, however it's sometimes simpler – and quicker – to take clues from what appears normally (without debugging). 

<https://github.com/helloSystem/ISO/issues/404#issuecomment-1256965602> for example, these two lines alone might have inspired someone to think about _localize_ in relation to _gnustep-back_: 

```text
[1/7] Fetching localize-0.0.1_11.pkg: 100%   33 KiB  34.1kB/s    00:01    
[2/7] Fetching gnustep-back-0.28.0_1.pkg: 100%  552 KiB 565.0kB/s    00:01    
```
